CHANGELOG.md

```diff
@@ -3,6 +3,59 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.15.0
+
+_Jul 28, 2022_
+
+We'd like to offer a big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Implement row pinning (#4863) @cherniavskii
+
+  Pro users can now [pin rows](https://mui.com/x/react-data-grid/rows/#row-pinning) on top or bottom of the grid.
+  To do so, enable the feature with `experimentalFeatures={{ rowPinning: true }}` and provide the row to pin with the `pinnedRows` prop.
+  For more details see the [documentation](https://mui.com/x/react-data-grid/rows/#row-pinning).
+
+- 🎁 Make aggregation public (#5592) @cherniavskii
+
+  Premium users can now add aggregated data in the grid.
+  Your user will be able to compute the average, sum, min, and other data with a single click.
```

Comment:
```suggestion
  You can now use aggregation functions, and with a couple of clicks, extract information like sum, average, count, and others
```
Took it from the aggregation blog post https://deploy-preview-33595--material-ui.netlify.app/blog/aggregation-functions/